model_compression_toolkit/gptq/keras/gptq_training.py

```diff
@@ -106,18 +113,56 @@ def __init__(self,
 
         self.weights_for_average_loss = self.compute_jacobian_based_weights(representative_data_gen)
 
-    def build_gptq_model(self):
+    def _is_gptq_applicable(self,
+                            node: common.BaseNode) -> bool:
+        """
+        A function for deciding if a layer should be fine-tuned during GPTQ.
+
+        Args:
+            node (BaseNode): Node for quantization decision
+
+        Returns:
+            A boolean whether the layer is to be wrapped with a QuantizeWrapper
+        """
+
+        if node.is_weights_quantization_enabled() and not self.fw_info.is_kernel_op(node.type):
+            common.Logger.error("GPTQ Error: Quantizing a node without a kernel isn't supported")
+        return node.is_weights_quantization_enabled()
```

Comment:
Just to confirm - GPTQ is only applicable to weights quantization?

Reply:
Yes, in GPTQ we only train the weights, we don't change the activation quantization solution.